Templates/HttpTrigger-CSharp/readme.md

```diff
@@ -0,0 +1,11 @@
+# HttpTrigger - C<span>#</span>
+
+The `HttpTrigger` makes it incredibly easy to have your functions executed via an HTTP call to your function.
+
+## How it works
+
+When you call the function, be sure you checkout which security rules you apply. If you're using an apikey, you'll need to include that in your request.
```

Comment:
"apikey" isn't our nomenclature is it? We have a "function key" that is specified in the url via a "code" query param, or x-functions-key header